controller/users.go

```diff
@@ -175,6 +194,11 @@ func (c *UsersController) Update(ctx *app.UpdateUsersContext) error {
 				return ctx.Conflict(jerrors)
 			}
 			identity.Username = *updatedUserName
+			keycloakUserProfile, err = c.copyExistingKeycloakUserProfileInfo(keycloakUserProfile, tokenString, accountAPIEndpoint)
```

Comment:
@sbose78 I see many calls to the function `c.copyExistingKeycloakUserProfileInfo`. I understand why you changed the code in this function. But why don't you move the call of the keycloak user profile to line 162 instead. Then we don't need to repeat this call again and again depending on the if else statements. If we get the token and accountAPIEndpoint and this function doesn't fail we can reuse the keycloak profile